target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -301,6 +302,9 @@ ath79_setup_macs()
 		lan_mac=$(fritz_tffs -n maca -i $(find_mtd_part "tffs (1)"))
 		wan_mac=$(fritz_tffs -n macb -i $(find_mtd_part "tffs (1)"))
 		;;
+    comfast,cf-wr752ac)
+		lan_mac=$(mtd_get_mac_binary art 0)
```

Comment:
As stated above, please report about the correct addresses. Eventually, you might put wan_mac here if device has a wan port.

Reply:
If this is a one-port as you said, just remove this case entirely.